### ISSUE TITLE
Change autogenerated Project ID

### ISSUE THREAD

**Issue by alneberg** [ScilifelabDataCentre/dds_cli#232]
I believe the 3-digit counter will be quickly run out of digits. Maybe it's not a problem? Or should we increase it to maybe 5 digits to at least save us some time?

**Comment by inaod568** [comment]
I agree, thought about this the other day. I will do this small fix.
